### PR TITLE
Changed styles into header component [Oleksii Savchenko]

### PR DESCRIPTION
**Pull request by Cestrus**
chenged styles. Add sort button

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -9,22 +9,36 @@ function Header(props) {
     return (
         <section className={styles.headerComponent}>
             <header className={styles.header}>
-                <Link to="/" className={'btn btn-primary'}>
+                <Link to="/" className={'btn btn-primary ' + styles.logoHeader}>
```
Comment (LordGemma) [inline src/components/Header/Header.jsx]: Please use template strings

**Comment by JIeCoRyI3** [comment]
Where we can see screens of new view???

**Comment by JIeCoRyI3** [comment]
As I remember, you need to do this: 
image
Why you dont leave a link to task in MR description? Follow Wiki please. 
And do all tasks following Trello.

**Comment by Cestrus** [comment]
header

File: src/components/SearchBar/SearchBar.jsx [hunk 2]
```diff
@@ -12,23 +12,23 @@ export default function SearchBar() {
             <div className={styles.filterBarContainer}>
                 <div className={styles.leftSide}>
                     <h5
-                        className={`${styles.leftSideItem} ${styles.searchTitle}`}
+                        className={`${styles.searchTitle} `}
```
Comment (LordGemma) [inline src/components/SearchBar/SearchBar.jsx]: In this case, a template string is not needed

**Comment by Cestrus** [comment]
restyle header, footer, main, not-found components
imd_2
imd_1

**Comment by JIeCoRyI3** [comment]
Header and Footer background color should be due to Wiki page. Seems like it is not. NotFound font color should be the same. Just remove stroke from NotFound.

**Pull request closed without merging**